network-store-model/src/main/java/com/powsybl/network/store/model/ConfiguredBusAttributes.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.store.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Map;
+
+/**
+ * @author Abdelsalem Hedhili <abdelsalem.hedhili at rte-france.com>
+ */
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@ApiModel("ConfiguredBus attributes")
+public class ConfiguredBusAttributes implements IdentifiableAttributes {
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @ApiModelProperty("Bus id")
+    private String id;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @ApiModelProperty("Bus name")
+    private String name;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @ApiModelProperty("ensure id unicity")
+    private boolean ensureIdUnicity;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @ApiModelProperty("voltage level id")
+    private String voltageLevelId;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @ApiModelProperty("bus voltage magnitude in Kv")
+    private double v;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @ApiModelProperty("voltage angle of the bus in degree")
+    private double angle;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @ApiModelProperty("active power in MW injected by equipments connected to the bus")
+    private double p;
```

Comment:
p and q should not be part of the model and data base as it is something that will be calculated from connected injections